ui/js/controllers/bugfiler.js

```diff
@@ -170,6 +170,14 @@ treeherder.controller('BugFilerCtrl', [
             // Only request the versions because some products take quite a long time to fetch the full object
             $.ajax("https://bugzilla.mozilla.org/rest/product/" + productString + "?include_fields=versions").done(function(productJSON) {
                 var productObject = productJSON.products[0];
+
+                // Versions where is_active is false can't be used
+                var i = 1;
+                while(productObject.versions[productObject.versions.length - i].is_active === false) {
+                    i = i + 1;
+                }
+                var version = productObject.versions[productObject.versions.length-i];
```

Comment:
You're after "the last version in the array that is active", right?

I think this will only work if all of the `is_active ==false` are right at the beginning, and will pick the wrong value if that's not the case.

It's also quite an unusual (and hard to follow) implementation, since:
* when iterating an array manually (ie not using lodash or similar), it's more common (and more readable) to use a `for` loop and break early, rather than the `while` usage here
* we want the last matching element, so we should be starting at the end of the array and working backwards, since we can break early
* ideally we shouldn't be hand-writing any of this, there are typically built-ins for these patterns (or at least polyfills in the form of lodash that can be used until we're happy to say an ES6 browser is required)

In fact, there is actually a lodash function that will do exactly what we're after:
https://lodash.com/docs#findLast

I think it would be good to also rename the variable `version` to something more accurately representing it's value, like `default_version`, `last_version`, or `most_recent_version` or similar. Moving the `XXX` comment below up to where this variable is set, would probably make things clearer too :-)